rr/gstreamer/gst_media.py

```diff
@@ -158,7 +169,7 @@ def get_triggers(self):
 
     @classmethod
     def make(cls, desc, all_triggers):
-        pipe = 'uridecodebin uri=%s caps=video/x-h264 ! queue ! h264parse ! avdec_h264 ! queue ! videoconvert ! videoscale ! video/x-raw,width=320,height=240,format=RGB ! appsink emit-signals=true name=appsink' % (
+        pipe = 'uridecodebin uri=%s caps=video/x-h264 ! queue ! h264parse ! v4l2h264dec capture-io-mode=5 ! video/x-raw,format=NV12 ! queue ! tiovxmultiscaler  src_0::pool-size=8 sink::pool-size=8 ! tiovxcolorconvert out-pool-size=8 ! video/x-raw,width=320,height=240,format=RGB ! queue ! appsink sync=true qos=false emit-signals=true drop=true max-buffers=3 name=appsink' % (
```

Comment:
It might be more readable to have this split in multiple lines:
```
pipe = '''uridecodebin uri=%s caps=video/x-h264 ! queue !
          h264parse ! v4l2h264dec capture-io-mode=5 ! video/x-raw,format=NV12 ! queue !
          tiovxmultiscaler  src_0::pool-size=8 sink::pool-size=8 !
          tiovxcolorconvert out-pool-size=8 ! video/x-raw,width=320,height=240,format=RGB ! queue !
          appsink sync=true qos=false emit-signals=true drop=true max-buffers=3 name=appsink''' % (
```